OpenRA.Game/Network/OrderManager.cs

```diff
@@ -169,19 +185,37 @@ void CheckSync(byte[] packet)
 				syncForFrame.Add(frame, packet);
 		}
 
-		public bool IsReadyForNextFrame => GameStarted && frameData.IsReadyForFrame(NetFrameNumber);
+		public bool IsReadyForNextFrame => GameStarted && IsReadyForFrame();
+
+		bool IsReadyForFrame()
+		{
+			return !ClientsNotReadyForFrame().Any();
```

Comment:
If this stays as the only use of `ClientsNotReadyForFrame` then IMO it would be clearer to get rid of that and use `return activeClients.All(client => pendingPackets[client].Count > 0);` here instead.